my-app/src/app/home/countdown-timer/countdown-timer.component.ts

```diff
@@ -0,0 +1,48 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'countdown-timer',
+  templateUrl: './countdown-timer.component.html',
+  styleUrls: ['./countdown-timer.component.css']
+})
+export class CountdownTimerComponent implements OnInit {
+  countDown = new Date('October 9, 2019 06:00:25').getTime();
+  timer = '';
+  now = new Date().getTime();
+  distance = this.countDown - this.now;
```

Comment:
You're using 2 fields here, while you're using 2 variables declared inside the setInterval function. You don't need both.